CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 set(LibraryVersion "1.0")
 add_definitions(-DLIBRARY_VERSION="${LibraryVersion}")
 
-COMPILELIB("")
+set(deps detector)
```

Comment:
I think that is conceptually wrong to add dependencies to any legacy class or library

Reply:
Yes, probably we just need the data members and the name of the class being preserved for data being read from previous ROOT files.